### ISSUE TITLE
[sub-issue] CRw: Alternate texts of all the Icons list are not readable

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#275]
![image](https://cloud.githubusercontent.com/assets/7510842/5356391/6d339180-7f9c-11e4-9a24-4dc10a925a9b.png)

All icons are read wrong by screen readers.
http://swiss.github.io/styleguide/base_layout_-_general.html#e-icons

==> Use ARIA "labelledby" or a hidden text.
If you use ARIA check it out with the screen reader NVDA http://www.nvaccess.org/download/

**Comment by tonifisler** [comment]
Main issue is #340

**Comment by tonifisler** [comment]
There is a paragraph about accessibility already here: http://swiss.github.io/styleguide/base_layout_-_general.html#e-icons

Note that the styleguide itself is a resource for developers: the elements are just a reference and the developer should be able to choose the correct `aria-` or `.sr-only` string depending on the case.

I made this paragraph more visible in an alert.
